ob/kustomize/overlay/7.3.0/defaults/kustomization.yaml

```diff
@@ -25,12 +25,6 @@ images:
 
 
 patches:
-  - target:
-      group: networking.k8s.io
-      version: v1
-      kind: Ingress
-      name: mtls
-    path: patch/ingress/mtls-ingress-patch.yaml
```

Comment:
What did this do??

Reply:
So we are using core ingress as the base, which has RCS/RS endpoints removed (although RS is being added back in now)

The kustomize files for OB will take the core ingresses, and apply patches to them to add routes not needed in core

If you look at patch/ingress/forgerock-ingress-patch.yaml you'll see how the rcs/ui route gets patched